src/ArduinoIoTCloudLPWAN.h

```diff
@@ -0,0 +1,87 @@
+/*
+   This file is part of ArduinoIoTCloud.
+
+   Copyright 2019 ARDUINO SA (http://www.arduino.cc/)
+
+   This software is released under the GNU General Public License version 3,
+   which covers the main part of arduino-cli.
+   The terms of this license can be found at:
+   https://www.gnu.org/licenses/gpl-3.0.en.html
+
+   You can be released from the requirements of the above licenses by purchasing
+   a commercial license. Buying such a license is mandatory if you want to modify or
+   otherwise use the software for commercial activities involving the Arduino
+   software without disclosing the source code of your own applications. To purchase
+   a commercial license, send an email to license@arduino.cc.
+*/
+
+#ifndef ARDUINO_IOT_CLOUD_LPWAN_H
+#define ARDUINO_IOT_CLOUD_LPWAN_H
+
+#include <ArduinoIoTCloud.h>
+#include <Arduino_ConnectionHandler.h>
+
+static uint8_t const DEFAULT_CBOR_LORA_MSG_SIZE = 255;
+
+class ArduinoIoTCloudLPWAN : public ArduinoIoTCloudClass {
+  public:
+    ArduinoIoTCloudLPWAN();
+    ~ArduinoIoTCloudLPWAN();
+    int connect();
+    bool disconnect();
+    int connected();
+    inline void update() {
+      update(NULL);
+    }
+    inline void update(int const reconnectionMaxRetries, int const reconnectionTimeoutMs) __attribute__((deprecated)) {
```

Comment:
I can't move `update()` and `update(int const reconnectionMaxRetries, int const reconnectionTimeoutMs)`  to `ArduinoIoTCloud ` because both call `update(CallbackFunc onSyncCompleteCallback)` that must be virtual... I've tried this solution:
```
ArduinoIoTCloud.h
inline void update() {
      update(NULL);
    }
    inline void update(int const reconnectionMaxRetries, int const reconnectionTimeoutMs) __attribute__((deprecated)) {
      update(NULL);
    }
virtual void update(CallbackFunc onSyncCompleteCallback) __attribute__((deprecated)) = 0;
```
but compiler can't resolve correctly the calling `ArduinoCloud.update()` unless I change the firm from `virtual void update(CallbackFunc onSyncCompleteCallback) __attribute__((deprecated)) = 0;`  to `virtual void update1(CallbackFunc onSyncCompleteCallback) __attribute__((deprecated)) = 0;` but for backward compatibility I can't